test/recursive_poseidon_to_snark.sh

```diff
@@ -33,9 +34,9 @@ node $RUNDIR/src/compressor12/main_compressor12_exec.js \
     -e $WORKSPACE/$FIRST_VERIFIER.exec \
     -m $WORKSPACE/$FIRST_VERIFIER.cm
 
-mkdir -p ./aggregation/$SECOND_VERIFIER/$INPUT_NUM
+mkdir -p ./aggregation/$SECOND_VERIFIER/
 
 ../target/release/eigen-zkit stark_prove -s ../starky/data/c12.starkStruct.bn128.json \
     -p $WORKSPACE/$FIRST_VERIFIER.pil.json \
     --o $WORKSPACE/$FIRST_VERIFIER.const \
-    --m $WORKSPACE/$FIRST_VERIFIER.cm -c $RUNDIR/circuits/$SECOND_VERIFIER.circom --i ./aggregation/$SECOND_VERIFIER/$INPUT_NUM/input.json --norm_stage 
\ No newline at end of file
+    --m $WORKSPACE/$FIRST_VERIFIER.cm -c $RUNDIR/circuits/$SECOND_VERIFIER.circom --i ./aggregation/$SECOND_VERIFIER/input.json --norm_stage
```

Comment:
It seems that this modification will lead to generating only one input.json because the output file of different circuits-inputs has the same position (./aggregation/$SECOND_VERIFIER/input.json)